test/testcafe/screenshot-test.js

```diff
@@ -0,0 +1,11 @@
+import { expect } from 'chai';
+import { statSync } from 'fs';
+
+fixture `Screenshot`
+    .page('https://google.com');
+
+test('Take screenshot', async t => {
+    var screenshotPath = await t.takeScreenshot();
```

Comment:
But, I'm not sure if we should document it anyway. For now can we rewrite test so it will use custom path?